pysteps/extrapolation/semilagrangian.py

```diff
@@ -109,9 +115,23 @@ def extrapolate(
 
     # defaults
     verbose = kwargs.get("verbose", False)
-    D_prev = kwargs.get("D_prev", None)
+    displacement_prev = kwargs.get("displacement_prev", None)
```

Comment:
I like the new keyword name. However, it may introduce backward compatibility problems. If the old **D_prev** keyword is used, it is ignored without any warning and the function behaves as if displacement_prev=None. One solution could be showing a warning message if the D_prev keyword is found in `**kwargs`, explaining that the keyword name was changed, and D_prev is being ignored.

Reply:
Good point. A warning message is now printed if D_prev is in `**kwargs`.